database/migrations/2016_03_22_124953_rename_phone_number_column_on_groups.php

```diff
@@ -12,7 +12,7 @@ class RenamePhoneNumberColumnOnGroups extends Migration
      */
     public function up()
     {
-        DB::statement('ALTER TABLE `groups` CHANGE phone_number phone tinyint(1)');
```

Comment:
Right, I tried that way and there was an issue with [renaming columns on a table that have an enum column](https://github.com/laravel/framework/issues/1186), even if it's not the enum column you're renaming. So I had to use a raw query here unfortunately.